Mathlib/SetTheory/Ordinal/Arithmetic.lean

```diff
@@ -382,8 +382,15 @@ theorem IsNormal.le_iff {f} (H : IsNormal f) {a b} : f a ≤ f b ↔ a ≤ b :=
 theorem IsNormal.inj {f} (H : IsNormal f) {a b} : f a = f b ↔ a = b := by
   simp only [le_antisymm_iff, H.le_iff]
 
+theorem IsNormal.id_le {f} (H : IsNormal f) : id ≤ f :=
+  H.strictMono.id_le
+
+@[deprecated IsNormal.id_le (since := "2024-09-11")]
 theorem IsNormal.self_le {f} (H : IsNormal f) (a) : a ≤ f a :=
```

Comment:
I actually think it makes more sense to have the applied variant as the main statement; I think it's pretty rare to use \le on functions. Using `id_le` everywhere means lean has to unfold the `id` again - from the other changes, it seems like it's happy to do that, though.

Reply:
I thought `id ≤ f` would be more idiomatic for a lemma literally called `id_le`. I sort of half-recall having been asked to change a similar applied lemma to the `Pi` version in the past. I'll follow whatever the consensus is on this.

Reply:
Uh yeah but the original lemma isn't called `id_le`?

Reply:
Alright, I'll keep `IsNormal.self_le` as is. What about `StrictMono.id_le` though?

Reply:
Can you make it follow the same naming convention? namely `id_le : id ≤ f`, `apply_le : n ≤ f n`

Reply:
Also yeah `self_le` should become `le_apply`

Reply:
Wait, but which one do I use? Do I have `id_le` and `le_apply` versions for both of these? Or do I only keep `StrictMono.id_le` and `IsNormal.le_apply`?

Reply:
I think you should have both versions for both